_sass/footer.scss

```diff
@@ -63,7 +63,7 @@
       }
 
       &.slack-icon:hover svg {
-        fill: #4b154b;
+        fill: #E01E5A;
```

Comment:
I appreciate the effort, @AnshumanDhiman. I'm not sure that this is an improvement over the current color, though.

Reply:
It's a good color. It's not apparent that it is an improvement over the existing color, though.

Reply:
Having said that... it's not any worse either.